src/tabmat/formula.py

```diff
@@ -24,6 +23,11 @@
 from .sparse_matrix import SparseMatrix
 from .split_matrix import SplitMatrix
 
+try:
+    from formulaic.materializers.base import EncodedTermStructure
+except ImportError:
+    from formulaic.materializers.types.formula_materializer import EncodedTermStructure
```

Comment:
Could we use the `formulaic` version to select the import path instead of a `try/except` block?

Reply:
Unfortunately I don't know in which version the change will appear. It's on main now, and my guess is `1.1`, but I'm not sure.